rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -1390,7 +1395,7 @@ object Reduce {
           toArgRaw   <- evalToLong(args(1))
           toArg      <- restrictToInt(toArgRaw)
           result     <- slice(baseExpr, fromArg, toArg)
-          _          <- costAccountingAlg.charge(sliceCost(toArg))
+          _          <- charge[M](sliceCost(toArg))
```

Comment:
AFAIK you shouldn't need the `[M]` type annotation in calls to `charge`.

Reply:
scalac disagrees